packages/netlify-cms-core/src/components/Collection/SortControl.js

```diff
@@ -13,6 +20,8 @@ const SortButton = styled(StyledDropdownButton)`
   &:after {
     top: 11px;
   }
+
+  color: ${props => (props.isActive ? colors.active : undefined)};
```

Comment:
@erezrokah The `color` style defined here is redundant and doesn't work when inline styling is not used. I think we can remove this. What do you say?

Reply:
You're right, forgot to remove it